pages/notifications.js

```diff
@@ -80,6 +86,8 @@ const Page = withT(({ url: { query, query: { context, token } }, t }) => {
     content = <TokenAuthorization
       email={email}
       token={token}
+      tokenType={tokenType || 'EMAIL_TOKEN'}
```

Comment:
Optional:
I've introduced `DEFAULT_TOKEN_TYPE` in Account/AuthSettings.js and use it for a similar fallback there - so you might wanna re-use that constant (or move it to e.g. components/constants.js).